app/src/main/res/drawable/ic_settings_network.xml

```diff
@@ -1,15 +1,17 @@
-<?xml version="1.0" encoding="utf-8"?>
-<vector
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="24dp"
     android:height="24dp"
     android:viewportWidth="24"
     android:viewportHeight="24">
-
+  <group>
+    <clip-path
+        android:pathData="M8,0L16,0A8,8 0,0 1,24 8L24,16A8,8 0,0 1,16 24L8,24A8,8 0,0 1,0 16L0,8A8,8 0,0 1,8 0z"/>
     <path
-        android:pathData="M13 3V10H19L11 21V14H5L13 3Z"
-        android:strokeWidth="2"
-        android:strokeColor="@color/textIconSecondary"
-        android:strokeLineCap="round"
-        android:strokeLineJoin="round" />
-</vector>
\ No newline at end of file
+        android:pathData="M0,0h24v24h-24z"
+        android:fillColor="#1DAEFF"/>
```

Comment:
here as well

Reply:
there no such color like this in UiKit, what should I do?

Reply:
ignore here then. 
Will talk to designers